apps/web/app/[language]/homestead/materials/page.tsx

```diff
@@ -0,0 +1,194 @@
+  const MetalTable = createDataTable(metalData.sources, ({ id }) => id);
+  
+  const woodData = getRefinedData(wood, WOOD_ID);
+  const WoodTable = createDataTable(woodData.sources, ({ id }) => id);
+  
+  const fiberData = getRefinedData(fiber, FIBER_ID);
+  const FiberTable = createDataTable(fiberData.sources, ({ id }) => id);
+
+  return (
+    <>
+      <Description>
+        <Trans id="homestead.materials.description"/>
+      </Description>
+      <Description>
+        <Trans id="homestead.materials.help"/>
+      </Description>
+
+      <label style={{ display: 'flex', gap: 16, alignItems: 'center' }}>
+        <Trans id="homestead.materials.efficiency"/>:
+        <Switch>
+          <Switch.Control type="link" replace active={efficiency === 1} href="/homestead/materials?efficiency=1">1</Switch.Control>
+          <Switch.Control type="link" replace active={efficiency === 2} href="/homestead/materials?efficiency=2">2</Switch.Control>
+          <Switch.Control type="link" replace active={efficiency === 3} href="/homestead/materials">3</Switch.Control>
+        </Switch>
+      </label>
+
+      <Headline id="refinedMetal" actions={<ColumnSelect table={MetalTable}/>}>
+        <ItemLink item={metalData.item}/>
+      </Headline>
+      <RefinedDataTable table={MetalTable}/>
+
+      <Headline id="refinedWood" actions={<ColumnSelect table={WoodTable}/>}>
+        <ItemLink item={woodData.item}/>
+      </Headline>
+      <RefinedDataTable table={WoodTable}/>
+
+      <Headline id="refinedWood" actions={<ColumnSelect table={FiberTable}/>}>
+        <ItemLink item={fiberData.item}/>
+      </Headline>
+      <RefinedDataTable table={FiberTable}/>
+
+      <PageView page="homestead/refinedMaterials"/>
+    </>
+  );
+}
+
+export const metadata = {
+  title: 'Homestead Refined Materials',
+};
+
+type RefinedDataSource = {
+  id: string;
+  item: Awaited<ReturnType<typeof getItems>>[string];
+  rate: ConversionRate;
+};
+type HomesteadRefinedMatsDataTable = {
+  Table: FC<DataTableProps<RefinedDataSource>>;
+  Column: FC<DataTableColumnProps<RefinedDataSource>>;
+  DynamicColumns: FC<DataTableDynamicColumnsProps<RefinedDataSource>>;
+  ColumnSelection: FC<DataTableColumnSelectionProps>;
+};
+
+const RefinedDataTable: FC<{ table: HomesteadRefinedMatsDataTable }> = ({
+  table,
+}) => (
+  <table.Table>
+    <table.Column id="id" title="Id" small hidden>
+      {({ id }) => id}
+    </table.Column>
+    <table.Column id="source" title="Source">
+      {({ item }) => <ItemLink item={item}/>}
+    </table.Column>
+    <table.Column
+      id="amountRequired"
+      title="Amount Required"
+      sortBy={({ rate }) => rate.required}
+      align="right"
+      hidden
+    >
+      {({ rate }) => rate.required}
+    </table.Column>
+    <table.Column
+      id="amountProduced"
+      title="Amount Produced"
+      sortBy={({ rate }) => rate.produced}
+      align="right"
+    >
+      {({ rate }) => rate.produced}
```

Comment:
Instead of a single column you can also wrap the item column with `OutputCount`.